lib/daru/io/exporters/csv.rb

```diff
@@ -5,16 +5,28 @@ module IO
     module Exporters
       module CSV
```

Comment:
What do you think about as we are starting a new gem, we'll drop all the "smart" and "meta" parts and go with simple classes? E.g. something which is used just like:
```ruby
exporter = Daru::IO::Exporter::CSV.new(args) # parses and validates arguments
exporter.call # actually exports
```
...and same for the importers? This way we can have, for example, proper inheritance of all importers from some base importer with service methods and stuff, and then just need to define some things like
```ruby
def to_csv(...)
  IO::Exporters::CSV.new(...).call
end
```
and similar methods (they even can be metaprogrammed to deduce proper importer name from method name).

This should be a great and immediate cleanup of IO mess.